stock_delivery_sale_section_and_note/models/stock_move.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+import datetime
+
+from odoo import fields, models
+
+from .stock_move_line import get_aggregated_properties
+
+
+class StockMove(models.Model):
+    _name = "stock.move"
+    _inherit = ["stock.move", "display.line.mixin"]
+
+    sequence = fields.Integer(related="sale_line_id.sequence", store=True)
```

Comment:
My concern is about this line.

It is highly tight on sale line presence. If the delivery is generated from another flow, how does this behave ?